### ISSUE TITLE
A normal quickfix list from :grep, :vimgrep, ... disappear when saving a buffer

### ISSUE THREAD

**Issue by nbl7** [Quramy/tsuquyomi#305]
When the plugin is installaed the normal quickfix list is not working anymore as expected.

Commands to reproduce the issue:

- :grep -R 'patterns' .
- :copen
- Navigate to a file from the quicklist created
- Save buffer with :w
- The quickfix list is empty now

**Comment by hoclun-rigsep** [comment]
Yeah, I too was wondering what users who use the quickfix list for other things too normally do about this.

**Comment by hoclun-rigsep** [comment]
Bump. Is there a way to use the location list instead, or some other workaround for preserving your quickfix?

At the least the quickfix should not change if there are no errors.
